pkg/segment/reader/segread/segreader/segreader.go

```diff
@@ -83,6 +112,57 @@ type SegmentFileReader struct {
 	allBmi             *structs.AllBlksMetaInfo
 }
 
+func GetBufFromPool(size int64) []byte {
```

Comment:
the GetBuf.. and PutBuf.. dont need to be public since they are not used outside of this module